free-programming-books.md

```diff
@@ -439,7 +437,6 @@
 * [Introduction to Data Science](https://docs.google.com/file/d/0B6iefdnF22XQeVZDSkxjZ0Z5VUE/edit?pli=1) - Jeffrey Stanton
 * [Introduction to Data Science](https://leanpub.com/datasciencebook) - Rafael A Irizarry *(Leanpub account or valid email requested)*
 * [Mining of Massive Datasets](http://www.mmds.org)
-* [School of Data Handbook](http://schoolofdata.org/handbook/)
```

Comment:
good url but uses cloudflare robot blocking.

Reply:
wow.. thats annoying.. didnt realize i had to manually click on that captcha